internal/types/backup.go

```diff
@@ -83,6 +83,8 @@ type Operation interface {
 	SetState(s OperationState)
 	GetMessage() string
 	SetMessage(m string)
+	SetCreatedAt(createdAt time.Time)
```

Comment:
i think we do not really need SetCreatedAt, GetCreatedAt in this interface. We only use them in restore handler => they can be fields of restore operation only